src/chatgpt-api.ts

```diff
@@ -400,21 +400,29 @@ export class ChatGPTAPI {
           }
         }, [] as string[])
         .join('\n\n')
```

Comment:
### Problem 2

I found that our calculations of prompt length in tokens have a mistake. 
It's -6 tokens diff when we have 1 systemMessage in the list (our estimation is 6 tokens less than real one from OpenAI)
It's -8 tokens diff when we have 2 messages in the list, regardless of the text in userPrompt
For other lengths it's bigger
But `_getTokenCount()` works correctly, because it calculates the length of response equally as OpenAI does.
It's something we `prompt ` string building. We need to check it (not in this PR)
Why do we add `Instructions:`, `User:` prefixes there? I didn't find any docs about it